### ISSUE TITLE
Single Prediction with Pre Trained Network inception v3 (Batch Norm Problem?)

### ISSUE THREAD

**Issue by b4shy** [keras-team/keras#12350]
Hi
I have a problem regarding single predictions when fine tuning the inception v3 net.
This is the code.
I already tried setting the learning phase to zero and one but the result remains the same.
I achieve great accuracy via model.eval()(about 95%+).
But a single predictions outputs about 50% accuracy on a binary problem

```
base_model = InceptionV3(weights="imagenet", input_shape = (224,224,3), include_top = False)
#set_untrainable(base_model)

x = base_model.output
x = GlobalAveragePooling2D()(x)
x = Dropout(0.5)(x)

x = Dense(2, activation="softmax")(x)

model = Model(inputs = base_model.input, outputs = x) 
```
Does anybody run into the same problem?

**Comment by b4shy** [comment]
Solved:
I did not convert the single image to a numpy array before applying preprocess_input.
This led the image to being all "-1" and therefore destroyed the prediction
